devtools/cli/src/main/java/io/quarkus/cli/QuarkusCli.java

```diff
@@ -185,6 +185,11 @@ public Optional<String> checkMissingCommand(CommandLine root, String[] args) {
             return Optional.empty();
         } catch (UnmatchedArgumentException e) {
             return Optional.of(args[0]);
+        } catch (Exception e) {
+            // For any other exceptions (e.g. MissingParameterException), we should just ignore.
```

Comment:
Should we somehow log.debug this for debugging purposes?

Reply:
I don't think so.

We really want to only catch the `UnmatchedArgumentException` at this stage.
Any other problem will be caught when Picocli is doing the same call a bit later (and then you have a proper error, not an obscure stracktrace).